docs/api/ptk_shell/formatter.rst

```diff
@@ -0,0 +1,10 @@
+.. _xonsh_ptk2_shell:
+
+******************************************************
+Prompt Toolkit 2+ Shell (``xonsh.ptk_shell.shell``)
```

Comment:
The title here should include that it is for the formatter

Reply:
done.